app/src/main/java/com/courseproject/tindar/usecases/chat/ChatInteractor.java

```diff
@@ -0,0 +1,99 @@
+package com.courseproject.tindar.usecases.chat;
+
+import com.courseproject.tindar.ds.DatabaseHelper;
+import com.courseproject.tindar.entities.MessageModel;
+
+import java.util.ArrayList;
+
+public class ChatInteractor implements ChatPresenter, ChatActivityController{
```

Comment:
ChatInteractor implements ChatInputBoundary (interface for that) and ChatPresenter and ChatActivityController should be dependency injected when instantiating ChatInteractor